validator/src/validator.ts

```diff
@@ -1,13 +1,19 @@
 import dotenv from "dotenv";
+import { privateKeyToAccount } from "viem/accounts";
 import { z } from "zod";
+import type { Participant } from "./consensus/storage/types.js";
 import { createValidatorService } from "./service/service.js";
-import type { ConsensusConfig } from "./types/interfaces.js";
+import type { ProtocolConfig } from "./types/interfaces.js";
 import { validatorConfigSchema } from "./types/schemas.js";
 
 dotenv.config();
 
+const BLOCKTIME_IN_SECONDS = 5n; // value assumed for gnosis chain
```

Comment:
I would create a `chainConfig.ts` that contains the chain ID and the block information - otherwise this might get accidentally out of sync once we use different networks.

Not for this PR, but a future improvement.